src/containers/home/index.tsx

```diff
@@ -39,14 +82,76 @@ const Home: React.FC = () => {
       </Helmet>
       <PageLayout>
         <HomeContainer>
-          <PageHeader
-            pageTitle={`${HOME_TITLE}${userName}!`}
-            pageSubtitle={HOME_HEADER}
-            subtitleColor={DARK_GREY}
-          />
-
-          <StyledSubtitle>Quick Links</StyledSubtitle>
-          <LinkCarousel />
+          {(() => {
+            switch (windowType) {
+              case WindowTypes.Mobile:
+                return (
+                  <>
+                    <PageHeader pageTitle={greeting} />
+                    <StyledSubtitle>Quick Links</StyledSubtitle>
+                    <LinkCarousel data={links} slidesPerPage={1} />
+                  </>
+                );
+              case WindowTypes.Tablet:
+                return (
+                  <>
+                    <PageHeader pageTitle={greeting} />
+                    <StyledSubtitle>Quick Links</StyledSubtitle>
+                    <LinkCarousel data={links} slidesPerPage={2} />
+                  </>
+                );
+              case WindowTypes.NarrowDesktop:
+                return (
+                  <>
+                    <PageHeader
+                      pageTitle={greeting}
+                      pageSubtitle={HOME_HEADER}
+                      subtitleColor={DARK_GREY}
+                    />
+                    <StyledSubtitle>Quick Links</StyledSubtitle>
+                    <List
+                      dataSource={links}
+                      grid={{ gutter: 1, column: 3 }}
+                      pagination={{
+                        pageSize: 3,
+                      }}
+                      renderItem={(item: LinkCardProps) => (
+                        <LinkCard
+                          text={item.text}
+                          path={item.path}
+                          background={item.background}
+                        />
+                      )}
+                    />
+                  </>
+                );
+              case WindowTypes.Desktop:
+                return (
+                  <>
+                    <PageHeader
+                      pageTitle={greeting}
+                      pageSubtitle={HOME_HEADER}
+                      subtitleColor={DARK_GREY}
+                    />
+                    <StyledSubtitle>Quick Links</StyledSubtitle>
+                    <List
```

Comment:
⛏️ Is there a way to move the pagination here so that its margins are aligned with the linkcard? Right now the right arrow is hanging off to the right

Reply:
I don't think there's an easy way to do this with the AntD component - can open a ticket for later to look into that.